_scicomputing/software_R.md

```diff
@@ -34,24 +34,50 @@ When using R/RStudio locally, you have the option to install a number of differe
+If computing resources beyond what is available via your desktop are required, you may consider running R scripts from the `rhinos` or `gizmo`.  `SciComp` makes pre-built R modules available for your use in order to facilitate more reproducible and reliable use of software on the local cluster.  
 
 ### Current R Modules on `Rhino`/`Gizmo`
-SciComp maintains a range of various builds of R on `Rhino` and `Gizmo` for use by researchers. Each build has different packages installed and versions of R itself, thus identifying if an existing R build matches your needs is a first step to using R on `Rhino` or `Gizmo`.  Specific information about which R Modules are available, including more information about packages installed in them can be found on our dedicated [R Module page](/rModules/). If you do not see the software you are looking for, email `scicomp` to request it or add your own GitHub issue in the [easybuild-life-sciences repo](https://github.com/FredHutch/easybuild-life-sciences).  Either way, please be specific about the source and version of the software you are interested in.  
-
+SciComp maintains a range of various builds of R on `Rhino` and `Gizmo` for use by researchers. Each build has different packages installed and versions of R itself, thus identifying if an existing R build matches your needs is a first step to using R on `Rhino` or `Gizmo`.  Specific information about which R Modules are available- including more information about packages installed in them- can be found on our dedicated [R Module page](/rModules/).  If you do not see the software you are looking for, email `scicomp` to request it or add your own GitHub issue in the [easybuild-life-sciences repo](https://github.com/FredHutch/easybuild-life-sciences).  Either way, please be specific about the source and version of the software you are interested in.
 
 ### `Rhino`
-Depending on what OS is on your local computer, the steps are:
-- If your local computer runs Windows, you should connect to the `Rhino`'s with [NoMachine](/scicomputing/access_methods/#nomachine-nx-access-multi-os), then launch RStudio from within NoMachine.
-- If your local computer is a Mac, you should [install XQuartz and connect to Rhino](/scicomputing/access_methods/), then launch RStudio from the terminal
-- If your local computer runs Linux, you simply need to connect to Rhino [with X11 forwarding](/scicomputing/access_methods/) using the `-X` flag as detailed above for Mac computers.
+
+You can connect to a `rhino` machine either with `ssh` (use `PuTTY` on Windows) or NoMachine. Once on a `rhino` machine, choose a version of R. You can see a list of R versions available with the command
+
+```
+module spider R
+```
+
+Choose one and invoke it with `ml`.  For example:
+
+```
+ml R/3.6.2-foss-2016b-fh1
+```
+
+Note that you can use tab-completion with the above command. For example, you can type
```

Comment:
Eventually might be nice to pull out `module` and tab-completion out into its own page.  Having it here seems a bit of a distraction to the main point of the page.

Don't need to fix in this PR tho...